examples/v2/doom/controller_test.go

```diff
@@ -69,21 +69,22 @@ func TestDoomController(t *testing.T) {
 	dc.SendKey("e")
 	dc.SendKey("e")
 	require.Equal(t, []string{
-		"w", "a", "e",
+		"w", "a", "e", "e", "e", "e",
```

Comment:
this shouldn't happen

if there is a broken test, please let me know.  this was ... well its a long story :)

fixed in master.

if you could revert these changes and we can possibly add the rest.